tests/chocolatey-tests/chocolatey.Tests.ps1

```diff
@@ -390,4 +390,20 @@ Describe "Ensuring Chocolatey is correctly installed" -Tag Environment, Chocolat
             $Output.Lines | Should -Contain '.NET 4.8 is not installed or may need a reboot to complete installation.'
         }
     }
+
+    Context 'Chocolatey lib directory missing' {
+        BeforeAll {
+            New-ChocolateyInstallSnapshot
+            Remove-Item -Path $env:ChocolateyInstall/lib/ -Recurse -Force
+            $Output = Invoke-Choco

```

Comment:
From memory, this is an incomplete test. The reason why an empty non-existing lib directory fails is because of NuGet itself when it tries to list packages that are installed.

Just running `choco` in this case would not trigger this happening (or at least not that I have seen).

The test should be a call to `choco list -lo` or something similar to that.

Reply:
Good catch. Apparently `choco` always exits 1, so this test wouldn't actually work. I've enhanced the test, but as per Slack discussions this PR will stay draft while we work through Licensed Extension behaviour.